docs/index.rst

```diff
@@ -1,11 +1,10 @@
 .. _index:
 
-=====================
-CrateDB Cloud Console
-=====================
+=======================
+CrateDB Cloud Reference
+=======================
 
-The *Cloud Console* is a hosted web administration interface for interacting
-with `CrateDB Cloud`_.
+TO DO: Add into paragraph here!
```

Comment:
you need to _TO DO_ this :)

Reply:
lol I was just thinking to keep the PR small. But I've added a paragraph :)